bahmnicore-omod/src/main/java/org/bahmni/module/bahmnicore/web/v1_0/controller/BahmniDrugOrderController.java

```diff
@@ -154,13 +155,13 @@ private void populateDrugConcepts(Concept concept, Set<Concept> drugConcepts) {
 
     private List<BahmniDrugOrder> getActiveOrders(String patientUuid, Date startDate, Date endDate) {
         List<DrugOrder> activeDrugOrders = drugOrderService.getActiveDrugOrders(patientUuid, startDate, endDate);
-        logger.info(activeDrugOrders.size() + " active drug orders found");
+        logger.info("active drug orders found {}", activeDrugOrders.size());
         return getBahmniDrugOrders(patientUuid,activeDrugOrders);
     }
 
     private List<BahmniDrugOrder> getPrescribedOrders(List<String> visitUuids, String patientUuid, Boolean includeActiveVisit, Integer numberOfVisits, Date startDate, Date endDate, Boolean getEffectiveOrdersOnly) {
         List<DrugOrder> prescribedDrugOrders = drugOrderService.getPrescribedDrugOrders(visitUuids, patientUuid, includeActiveVisit, numberOfVisits, startDate, endDate, getEffectiveOrdersOnly);
-        logger.info(prescribedDrugOrders.size() + " prescribed drug orders found");
+        logger.info("prescribed drug orders found {}", prescribedDrugOrders.size());
```

Comment:
not a big deal but the sentence would read confusing to put the number at the end...